### ISSUE TITLE
Cannot set background color for multiple cells in Firefox

### ISSUE THREAD

**Issue by chscott** [ckeditor/ckeditor-dev#2439]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Launch the current version of Firefox (I'm on 62.0.2 64-bit)
2. Create a table at the demo site https://ckeditor.com/ckeditor-4/ (default options)
3. Select two cells and choose Cell -> Cell Properties. Enter a color in the Background Color field (I used "yellow").

### Expected result

Both selected cells have a yellow background.

### Actual result

Only the first cell has a yellow background. The same steps work as expected in Chrome and IE.

## Other details

* Browser: Firefox 62.0.2 64-bit
* OS: Windows 7
* CKEditor version: 4.10.1 (Standard) 
* Installed CKEditor plugins: Whatever plugins are installed on the demo site

**Comment by jacekbogdanski** [comment]
Hello,

I'm not able to reproduce the issue (tried on Windows and Linux) but everything works fine.

![peek 2018-09-26 23-52](https://user-images.githubusercontent.com/10279586/46111407-2d838400-c1e7-11e8-9983-0523dcf21024.gif)

Please, provide more information about the issue if you still are able to reproduce it.

**Comment by chscott** [comment]
@jacekbogdanski When selecting cells, make sure your mousedown event occurs outside the cell/table border.

**Comment by chscott** [comment]
But why does this work in Chrome and IE (and versions of Firefox prior to 62)?

**Comment by jacekbogdanski** [comment]
You are right, I checked this issue again and it seems like it's working correctly between different browsers except for Firefox. It may be Gecko upstream bug but it would be nice to make sure if it cannot be handled by CKEditor.
